test/conditions/text_test.exs

```diff
@@ -20,23 +20,34 @@ defmodule FiltrexConditionTextTest do
     })
   end
 
+  test "encoding map value" do
+    text = "Lorem Ipsum"
+    assert Filtrex.Encoders.Map.encode_map_value(condition("equals", text)) == text
+  end
+
   test "encoding as SQL fragments for ecto" do
     {:ok, condition} = Text.parse(@config, %{inverse: false, column: "title", value: "Buy Milk", comparator: "equals"})
-    encoded = Filtrex.Encoder.encode(condition)
+    encoded = Filtrex.Encoders.Fragment.encode(condition)
```

Comment:
Because this is module is quite long, can we alias these references in tests at the top of the file:

```elixir
alias Filter.Encoders.Fragment
```

Usage:
```suggestion
    encoded = Fragment.encode(condition)
```